src/accidental.ts

```diff
@@ -17,38 +17,75 @@ import { Modifier } from './modifier';
 import { Glyph } from './glyph';
 import { GraceNoteGroup } from './gracenotegroup';
 import { GraceNote } from './gracenote';
+import { ModifierContextState } from './modifiercontext';
+import { Voice } from './voice';
+import { Note } from './note';
+import { StaveNote } from './stavenote';
+
+export interface Line {
```

Comment:
This can also be an internal type alias.

Reply:
done